.drone.yml

```diff
@@ -106,6 +108,12 @@ platform:
   arch: amd64
 
 steps:
+  - name: Build theme files
+    image: node:current-slim
+    commands:
+      - npm install --global yarn
```

Comment:
I think this might install [Yarn v1](https://classic.yarnpkg.com/lang/en/docs/install/)? Project uses [Yarn v3](https://yarnpkg.com/getting-started/install).

Bit of a pain overall, not sure it's worth using v3. But yeah if we continue to I think this needs to change.

Reply:
I did try to promote it but it failed anyway because it allready exists ... 

https://drone.hestiacp.com/hestiacp/hestiacp/1922